extensions/wikia/PortableInfobox/services/Helpers/PortableInfoboxRenderServiceHelper.php

```diff
@@ -172,17 +176,21 @@ private function getThumbnail( $title ) {
 	 * if $wgPortableInfoboxCustomImageWidth is set, do not change max height
 	 * and set width to $wgPortableInfoboxCustomImageWidth.
 	 * @param $image
+	 * @param string $theme
 	 * @return array width and height
 	 */
-	public function getImageSizesForThumbnailer( $image ) {
+	public function getImageSizesForThumbnailer( $image, $theme ) {
 		global $wgPortableInfoboxCustomImageWidth;
 
+		$desktopThumbnailWidth = $theme === self::EUROPA_THEME ? self::EUROPA_THUMBNAIL_WIDTH :
```

Comment:
Parenthesis still make those lines easier to read. I also recommend extracting to getThumbnailWidth($theme)